revocation.md

```diff
@@ -0,0 +1,237 @@
+# Revocation
+
+**Patent Pending**
+
+Revocation requires that a verifiable revocation notice for a given Doughnut be posted somewhere
+accessible to all Doughnut revokers and verifiers.
+
+Because Doughnuts support decentralisation it makes sense for a Doughnut to tell verifiers
+how and where to revoke it, and where to check its revocation status.
+This removes the need for centralised revocation services.
+
+What follows are specifications for decentralised Doughnut revocation methods.
+A Doughnut version may specify a specific revocation method, or allow the issuer to
+make a choice of revocation method.
+
+Different strategies for achieving revocation are explored in the [`Strategies`](#strategies) section below.
```

Comment:
We should somewhere mention in the original doughnut paper mention that a doughnut is valid only if it's not revoked. In particular, at the time of checking the validity of a doughnut, **if the revocator is not accessible** by the entity who is processing the doughnut, **it should be deemed as invalid.**